documentation/docs/install-pmm/plan-pmm-installation/hardware_and_system.md

```diff
@@ -1,55 +1,64 @@
+    - **Storage**: 500+ GB
+
+## Storage planning
 Adjust storage calculations based on your data retention period and the number of metrics collected. To estimate storage requirements:
 
-- allow approximately 1 GB of storage per monitored node per week.
-- for the default 30-day retention period, use the formula: `number_of_nodes * 4 GB`.
+- **Base formula**: `nodes × retention_period_in_weeks × 1 GB`
+- **Quick estimate**: for the default 30-day retention period, use the formula `number_of_nodes x 4 GB`
+- **High-precision monitoring**: increase estimates by 20-50% when using 1-second collection intervals
 
 ### Server architecture requirements
 
-- **CPU**: Must support the [`SSE4.2`](https://wikipedia.org/wiki/SSE4#SSE4.2), which is required for Query Analytics (QAN).
-- **ARM64**: Ensure your system uses a supported ARM64 architecture (e.g., ARMv8).
+- **CPU**: must support the [`SSE4.2`](https://wikipedia.org/wiki/SSE4#SSE4.2), which is required for Query Analytics (QAN).
+- **ARM64**: ensure your system uses a supported ARM64 architecture (e.g., ARMv8).
 - **ARM limitations**: PMM Server is not currently available as a native ARM64 build. For ARM-based systems, use Docker or Podman to run x86_64 images via emulation.
 
-!!! hint alert alert-success "Tip"
-    To reduce storage usage, consider [disabling table statistics](../../optimize/disable_table_stats.md), which can significantly decrease the size of the VictoriaMetrics database.
+### Storage optimization
+
+To reduce storage usage, consider [disabling table statistics](../../install-pmm/install-pmm-client/connect-database/mysql/disable_table_stats.md), which can significantly decrease the size of the VictoriaMetrics database.
 
-## Client requirements
+## PMM Client requirements
 
 ### Storage
 
-The PMM Client package requires 100 MB of storage for installation. Under normal operation with a stable connection to PMM Server, no additional storage is needed. During network instability or low throughput periods, the Client temporarily stores collected data that cannot be immediately dispatched. The VM Agent reserves 1 GB of disk space for caching during network outages, while Query Analytics (QAN) utilizes RAM instead of disk storage for its cache.
+The PMM Client package requires 100 MB of storage for installation. Under normal operation with a stable connection to the PMM Server, no additional storage is needed. However, during network instability or low throughput periods, the Client temporarily caches collected data until it can be transmitted. 
+
+The VM Agent reserves 1 GB of disk space for caching during network outages, while Query Analytics (QAN) uses RAM instead of disk storage for its cache.
+
+### Operating system compatibility
 
-### Operating system
+PMM Client is compatible with modern 64-bit Linux distributions on both x86_64 and ARM64 architectures. Supported platforms include current versions of Debian, Ubuntu, CentOS, and Red Hat Enterprise Linux. 
```

Comment:
Shall we mention Oracle Linux as it's our base OS in docker? Also, shall we say `"Red Hat" derivatives` instead of mentioning just two?